### PR TITLE
(feat): updated the functionality of the imaging order

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Add Age,  Gender and clinical diagnosis as part of  the patient info on the imaging panel
- Review of the imaging procedure before approval
- Printing of the results
- Updated the UI

## Screenshots
imaging

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by ojwanganto** [comment]
Thanks so much, @its-kios09. I am wondering whether we can make the report body showing the report and other details a bit bigger and centralized?

**Comment by its-kios09** [comment]
> Thanks so much, @its-kios09. I am wondering whether we can make the report body showing the report and other details a bit bigger and centralized? I just googled and got the below sample. Do you think we can achieve something similar or close?
> 
> image

Yes, let me design the report then i will share it with you

File: packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+@use '@carbon/type';
+@use '@carbon/colors';
+
+.widgetContainer {
+  padding: layout.$spacing-03;
+}
+
+.actionButton {
+  --cds-layout-size-height-context: var(--cds-layout-size-height-sm, 2rem);
+  --cds-layout-size-height: var(--cds-layout-size-height-context);
```
Comment (donaldkibet) [inline packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss:11]: why use this and we have the tokens for length e.g `layout.$spacing-05`

File: packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  bottom: layout.$spacing-05;
+  left: 4%;
+  width: 80%;
+  display: flex;
+  gap: layout.$spacing-02;
+
+  & > button {
+    max-width: 40%;
+    width: 40%;
+  }
```
Comment (donaldkibet) [inline packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss:38]: Is this working well on tablets

**Review by donaldkibet — CHANGES_REQUESTED** [review]
[Kapture 2024-12-17 at 10.25.29.webm](https://github.com/user-attachments/assets/bda200c8-936d-4913-a1da-d9b2750f4fcd)

Am getting this error on running this PR

**Comment by donaldkibet** [comment]
cc @its-kios09 see the conflicts

File: packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss (hunk 2 above)
Comment (its-kios09) [reply packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss:38]: Addressed

File: packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss (hunk 1 above)
Comment (its-kios09) [reply packages/esm-imaging-orders-app/src/form/review-form/review-imaging-form.scss:11]: sorted

**Review by donaldkibet — APPROVED** [review]

**Comment by ojwanganto** [comment]
Do we really need the background colors for order status and no instructions? Could we just adopt a UI that's not very shouting? Disclaimer: Am not a UI expert :)
We can discuss this for the next iteration

**Comment by donaldkibet** [comment]
Sure @ojwanganto, Merging this in

**Pull request merged**